tools/downstream-jenkins-sync-jobs.yml

```diff
@@ -0,0 +1,53 @@
+---
+# This playbook is used to sync the jenkins jobs from one ocp pod folder to another pod folder
+# Usage:
+# ansible-playbook downstream-jenkins-sync-jobs.yml --extra-vars "local_tmp_folder=~/jenkins_jobs src_pod=gluster-downstream-jenkins src_folder=/var/lib/jenkins/restore/jobs/ dest_pod=gluster-new-jenkins dest_folder=/var/lib/jenkins/jobs/"
+# Varibles:
+# local_tmp_folder - Local folder that is created temporarily by the playbook for storing the jobs during transfer between the source pod to the destination pod by default its /tmp/sync_jobs
+# src_pod - The pod name that holds the jobs that will get copied to the destination pod
+# src_folder - The folder on the src_pod that includs the jobs the will be copied to the dest_pod
+# dest_pod - The pod name that the jobs will get copied to
+# dest_folder - The folder on the dest_pod where the jobs the will be copied to
+# 
+- hosts: localhost
+  gather_facts: false
+  vars:
+      local_tmp_folder: "/tmp/sync_jobs"
+  tasks:
+
+  - name: Check oc tool installation status
+    command: which oc
+    changed_when: false
+    failed_when: false
+    register: oc_installed
+
+  - name: Fail if oc tool is not installed
+    fail:
+      msg: "oc tool appears to be missing, install first and connect with your user to the ocp cluster by running oc login"
+    when: oc_installed is failed
+
+  - name: Check connected oc client user
+    command: oc whoami
+    ignore_errors: True
+    register: oc_whoami
+
+  - name: Fail if oc user is not connected
+    fail:
+      msg: "Please login to the ocp cluster by running oc login"
+    when: oc_whoami.rc != 0
+
+  - name: Create local directory
+    file:
+      path: "{{ local_tmp_folder }}"
```

Comment:
Maybe just have the playbook create this directory instead of having to manually create and then pass it as a variable?

`mktemp -d`

Reply:
Changed it so that now it creates a tmp dir without passing a variable

Reply:
:+1: good work!

Reply:
Thanks man!